yarn.lock

```diff
@@ -221,6 +221,11 @@
   resolved "https://registry.yarnpkg.com/@zeit/schemas/-/schemas-2.6.0.tgz#004e8e553b4cd53d538bd38eac7bcbf58a867fe3"
   integrity sha512-uUrgZ8AxS+Lio0fZKAipJjAh415JyrOZowliZAzmnJSsf7piVL5w+G0+gFJ0KSu3QRhvui/7zuvpLz03YjXAhg==
 
+abbrev@1:
```

Comment:
@jvanbaarsen not sure you meant to commit the yarn.lock file?

Reply:
I have the same yes, but it should still be a separate commit/PR

Reply:
I'll revert this one, and do it in a different branch then.